dspace-api/src/main/java/org/dspace/uclouvain/xmlworkflow/actions/NotifyAuthorsAction.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.uclouvain.xmlworkflow.actions;
+
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dspace.access.status.DefaultAccessStatusHelper;
+import org.dspace.access.status.service.AccessStatusService;
+import org.dspace.content.Bitstream;
+import org.dspace.content.Bundle;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Constants;
+import org.dspace.core.Context;
+import org.dspace.core.Email;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.dspace.uclouvain.core.mail.MailMetadataParserService;
+import org.dspace.xmlworkflow.state.Step;
+import org.dspace.xmlworkflow.state.actions.ActionResult;
+import org.dspace.xmlworkflow.state.actions.processingaction.ProcessingAction;
+import org.dspace.xmlworkflow.storedcomponents.XmlWorkflowItem;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Action to notify all the authors of a publication when it is deposited.
+ *
+ * @author Michaël Pourbaix <michael.pourbaix@uclouvain.be>
+ */
+public class NotifyAuthorsAction extends ProcessingAction {
+
+    private static final List<String> FIELDS_TO_EXPOSE = Arrays.asList(
+        "dc.type.maintype",
+        "dc.title",
```

Comment:
you can add 'dateIssued'; the best solution should be to have a confirugation into `uclouvain.cfg`